pom.xml

```diff
@@ -88,11 +88,6 @@
 			<artifactId>json-path-assert</artifactId>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.junit.platform</groupId>
-			<artifactId>junit-platform-commons</artifactId>
-			<version>RELEASE</version>
-		</dependency>
```

Comment:
We do not need this dependency anymore?

Reply:
Nope, it's not being used since we have apache commons